core/domain/learner_progress_services.py

```diff
@@ -398,29 +489,35 @@ def get_all_completed_exp_ids(user_id):
         return []
 
 
-def get_completed_exp_summaries(user_id):
+def get_filtered_completed_exp_summaries(exploration_summaries,
+                                         exploration_ids):
     """Returns a list of summaries of the completed exploration ids and the
     number of explorations deleted from the list as they are no longer present.
```

Comment:
@Arunabh98 did you miss this comment? It's no longer the *number* of explorations that are being returned...